### PR TITLE
feat: implement request: checkFileStatus

### PR DESCRIPTION
**Pull request by TerminalFi**

### CODE REVIEW

File: plugin/commands.py [hunk 1]
```diff
@@ -263,6 +266,17 @@ def _on_result_check_status(self, payload: CopilotPayloadSignInConfirm | Copilot
             message_dialog("You haven't signed in yet.")
 
 
+class CopilotCheckFileStatusCommand(CopilotTextCommand):
+    @_provide_plugin_session()
+    def run(self, plugin: CopilotPlugin, session: Session, _: sublime.Edit) -> None:
+        file_path = file_path = self.view.file_name() or ""
```
Comment (jfcherng) [inline plugin/commands.py:272]: duplicate?
`file_path = file_path` = self.view.file_name() or ""

**Review by jfcherng — CHANGES_REQUESTED** [review]

**Review by jfcherng — APPROVED** [review]
LGTM!

**Comment by jfcherng** [comment]
I have an idea that this information can be always shown in the status bar text.

**Comment by TerminalFi** [comment]
I'm working on it respecting git ignore so this might be good

**Pull request merged**